services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchPortConfigSpec.groovy

```diff
@@ -46,7 +47,7 @@ class SwitchPortConfigSpec extends BaseSpecification {
             assert islUtils.getIslInfo(islUtils.reverseIsl(isl)).get().state == IslChangeType.FAILED
         }
 
-        and: "Port failure is logged in OTSDB"
+        and: "Port failure is logged in OpenTSDB"
         def statsData = [:]
         Wrappers.wait(WAIT_OFFSET) {
             statsData = otsdb.query(portDownTime, "pen.switch.state",
```

Comment:
May be a good idea for the future refactoring to move OpenTSDB metrics names to a designated place (as there were some discussions about possible renaming of them)

Reply:
I have pretty much the same opinion as with moving out the error messages: we can/will do it if majority will decide so, but my vote will go for keeping test data as close to the test itself as possible (since these strings will never be reused anywhere else. Otherwise, of course moving them out is a must)